QuizWhois.Domain/Services/Implementations/QuestionRatingService.cs

```diff
@@ -47,6 +50,7 @@ public QuestionRatingModel AddRating(long questionModelId, long userId, uint rat
             var entity = new QuestionRating(questionModelId, userId, rating);
```

Comment:
I would like to see logs for all the errors that could occur, nontheless there are no LogError calls